docs/_docs/installation.md

```diff
@@ -15,9 +15,9 @@ Before you start, make sure your system has the following:
 
 - GNU/Linux, Unix, or macOS
 - [Ruby](https://www.ruby-lang.org/en/downloads/) version 2.2.5 or above, including all development
-  headers
-- [RubyGems](https://rubygems.org/pages/download)
-- [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/) (in case your system doesn't have them installed, which you can check by running `gcc -v` and `make -v` in your system's command line interface)
+  headers (ruby installation can check by running `ruby -v`, development headers can be checked on Ubuntu by running `apt list --installed  ruby-dev`)
```

Comment:
```diff
- (ruby installation can check by running `ruby -v`,
+ (ruby installation can check be by running `ruby -v`,
```

Reply:
Maybe it is better to make a Passive voice - "(ruby installation can be checked by running `ruby -v`"?

Reply:
of course.. I meant to type `(ruby installation can be checked by running ruby -v` itself :laughing: :man_facepalming:

Reply:
Done!  👍